### DISCUSSION TITLE
Enabling translation

### DISCUSSION THREAD

**Discussion by Marius-Ar** [winder/Universal-G-Code-Sender#2488] [Help] [answered]
Hello,

I've been looking for the ([french](https://github.com/winder/Universal-G-Code-Sender/blob/master/ugs-core/src/resources/MessagesBundle_fr_FR.properties)) translation feature for quite sometime now. I think I checked everywhere and did not find an answer. I came across Settings > Editor > Formatting > Language, but I guess we're talking "language" in the programming sense here? The list is empty anyways

I did a search on the [official website](https://universalgcodesender.com/) and on the Internet, but no result, except the "Contributing to Translation" page. It looks like it was not mentioned either in the "discussions" section of this repo.

Could anybody help me find this feature please? Other users might want this information as well, would be happy to contribute to the documentation if it's a shared opinion. Thanks! 😄

**Comment by breiler** [comment, marked as answer]
Do you mean how to switch language to french? Checkout this wiki page:
https://github.com/winder/Universal-G-Code-Sender/wiki/Configuration#language
